### ISSUE TITLE
[A11y]Narrator unnecessarily announces the word expanded for the Cancel button.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7878]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1062759

Steps to Reproduce:
1. Hit the #URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “API Keys” from the Profile Dropdown and hit enter.
5. Navigate to "Create" button and hit enter.
6. Enter Values in "Key Name" and "Glob Pattern" field and hit enter on Create button.
7. Navigate to "Manage" control and hit enter on "Edit" control.
8. Navigate to cancel button
9. Turn on the narrator 
10. Check if the narrator announces the cancel button correctly. 

Actual:
Narrator announces the name of the cancel button incorrectly. Narrator announces cancel button like "Cancel button expanded".  

Same issue is observed in chrome+NVDA

Similar issue has been observed in following:
1. Hit the URL: https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “API Keys” from the Profile Dropdown and hit enter.
5. Navigate to "Create" button and hit enter, Select Push Checkbox.
6. "Select Packages" section gets enabled.
7. Navigate to 'Cancel' button and check the announcement by narrator.

Expected:
Narrator should not announce expanded along with cancel button.  
Narrator should announce only Cancel button.

Notes:
I was unable to reproduce the behaviour in the second scenario.
In the first scenario, the button gets the aria-expanded="true" attribute, which causes narrator to read it as expanded. However, in that case, the button is a collapser for the expanded dialog (from edit). Not 100% sure what we should do here. Removing aria-expanded="true" from the element will "fix" this bug, but I'm not convinced that this is the correct behaviour.

**Comment by ryuyu** [comment]
It seems like another solution here is for us to just remove the cancel button altogether since it doesn't seem to serve any purpose other than being a second collapser.

**Comment by ryuyu** [comment]
I will remove the aria-expanded on it and treat it as a pure cancel button.

**Comment by ryuyu** [comment]
Turns out that the aria-expanded is actually put there by knock-out(?) when we do a data-toggle collapse. We cannot remove it.
